source/docs/type-properties.md

```diff
@@ -5,6 +5,110 @@ next: /docs/boolean-type.html
 ---
 # Common Type Properties
 
-> No content yet
+<a name="desc">
+## desc / description
+
+The `desc` (or `description`) property controls the text displayed immediately the right
+of the option or argument in the generated help text.
+
+If not specified, the description will be blank.
+
+```js
+sywac.boolean('--confirm', {
+  desc: 'skip confirmation prompt'
+});
+```
+```bash
```

Comment:
Probably doesn't make a difference, but I usually use `console` instead of `bash`.